rust/src/smb/files.rs

```diff
@@ -33,17 +35,32 @@ pub struct SMBTransactionFile {
     /// after a gap, this will be set to a time in the future. If the file
     /// receives no updates before that, it will be considered complete.
     pub post_gap_ts: u64,
+    pub file_range: *mut HttpRangeContainerBlock,
```

Comment:
@jasonish can I keep the derive Default and have `file_range` being null ?

Reply:
> @jasonish can I keep the derive Default and have `file_range` being null ?

No, I do not believe so.